src/Relation/Binary/HeterogeneousEquality.agda

```diff
@@ -233,3 +233,15 @@ data Reveal_is_ {a} {A : Set a} (x : Hidden A) (y : A) : Set a where
 inspect : ∀ {a b} {A : Set a} {B : A → Set b}
           (f : (x : A) → B x) (x : A) → Reveal (hide f x) is (f x)
 inspect f x = [ refl ]
+
+------------------------------------------------------------------------
+-- Heterogeneous proof irrelevance
+
```

Comment:
These two properties could have more meaningful names? Maybe something like `≅-irrelevance` to match `≡-irrelevance` in 'PropositionalEquality'.

Reply:
I've cleaned this up in the merge commit: https://github.com/agda/agda-stdlib/commit/e5ae09d93c8a87bc591bc468fb911b9e64ae3aa5#diff-a07ef1a4ca3de2d7c7136b83a5c15b34R96

Reply:
Btw, you can see the cumulated effects of all the commits on that branch here: https://github.com/agda/agda-stdlib/compare/quotients

Reply:
Ah, sorry hadn't seen that. Thanks for the link!